src/main/java/io/netty/incubator/codec/quic/QuicChannel.java

```diff
@@ -172,6 +172,14 @@ default ChannelPromise voidPromise() {
      */
     boolean isTimedOut();
 
+     /**
+      * Return the {@link SocketAddress} which the {@link Channel} is connected to.
+      *
+      * @return the {@link SocketAddress} which the {@link Channel} is connected to.
+      *         {@code null} if this channel is not connected
+      */
+    SocketAddress getRemote();
```

Comment:
Can we call it `SocketAddress#remoteAddress()`?

Reply:
Maybe, but I'm afraid that there may be confusion with the standard method `Channel#remoteAddress()`

Reply:
Then how about `SocketAddress#remoteNetworkAddress`?